x-pack/legacy/plugins/file_upload/public/legacy.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { npStart } from 'ui/new_platform';
+import { plugin } from '.';
```

Comment:
that looks weird

Reply:
Pulled from [the example](https://github.com/elastic/kibana/blob/master/src/core/MIGRATION.md#3-export-your-runtime-contract) on that one and found a few others who used the same syntax. No strong feelings either way personally so I just stuck with the example syntax